src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,41 @@
+.catalog {
+  display: grid;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: $gap;
+
+  @media (max-width: 488px) {
+    & {
+      grid-template-columns: repeat(1, $card-width);
+    }
```

`&` selector is redundant here
```suggestion
      grid-template-columns: repeat(1, $card-width);
```